firmware-management/updated/.clang-tidy

```diff
@@ -1,6 +1,6 @@
 ---
-Checks:          'clang-diagnostic-*,clang-analyzer-*,*,-clang-analyzer-*,clang-analyzer-cplusplus-*'
-WarningsAsErrors: 'clang-diagnostic-*,clang-analyzer-*,*,-clang-analyzer-*,clang-analyzer-cplusplus-*'
+Checks:          'clang-diagnostic-*,clang-analyzer-*,*,-clang-analyzer-alpha.*,-llvm-include-order,-modernize-use-trailing-return-type,-modernize-use-trailing-return-type,-modernize-concat-nested-namespaces,-fuchsia-default-arguments-calls'
```

Comment:
I think we should add the copyright notice and SPDX License identifier to this file.